cmd/login.go

```diff
@@ -57,13 +57,22 @@ func init() {
 	lgCmd.PersistentFlags().StringVar(&loginCmdArgs.aadEndpoint, "aad-endpoint", "", "the Azure active directory endpoint to use for OAuth user interactive login")
 	// Use identity which aligns to Azure powershell and CLI.
 	lgCmd.PersistentFlags().BoolVar(&loginCmdArgs.identity, "identity", false, "log in using virtual machine's identity, also known as managed service identity (MSI)")
+	// Use SPN certificate to log in.
+	lgCmd.PersistentFlags().BoolVar(&loginCmdArgs.certificate, "certificate", false, "log in using a certificate as a form of Service Principal Name (SPN) auth")
```

Comment:
These flags seem a bit redundant, right? We can infer that we are trying to use a certificate with the existed of cert-path and cert-pass

Reply:
Resolved in [@1c8b21](https://github.com/Azure/azure-storage-azcopy/pull/332/commits/1c8b217cd60befb694f9a063a5603dca8258958f), flags merged